mu-plugins/blocks/horizontal-slider/src/block.js

```diff
@@ -95,10 +95,12 @@ function Block( { items, title } ) {
 	const height = width * aspectRatio;
 
 	return (
-		<div>
+		<div aria-labelledby="sliderHeading">
```

Comment:
In the past I've been told that using the correct heading tags (like the H3 this references) is pretty effective for screen reader users, so I'm wondering if this is necessary... did you find there wasn't enough context without it?

Reply:
To be honest, I was following along with:
https://www.w3.org/WAI/tutorials/carousels/structure/

I'm totally fine with removing it.